### DISCUSSION TITLE
Using rome check alongside svelte-check?

### DISCUSSION THREAD

**Discussion by Ashvith10** [rome/tools#4722] [Q&A]
I'm currently working on a hobby project that uses Svelte and Typescript (not just limited to inline TS, I have a few TS files to separate the logic from the view). Should I be using rome check alongside svelte-check? And if so, what would be a good way to  configure them together?

Here's what I had in my mind:
```json
"scripts": {
    "check": "svelte-check --tsconfig ./tsconfig.json && rome check --apply-unsafe ./src/lib",
    "format": "rome format . --write"
}
```

**Comment by ematipico** [comment]
Not sure what svelte check does, but the Rome command like you suggested will apply all unsafe fixes. So make sure that svelte check does that too.

You can mix them together, but Rome will emit diagnostics for files that don't recognize e.g. `.svelte` files.

You could use the nightly release, which exposes new flags to not raise diagnostics on unknown files.
